modules/nw-ingress-sharding.adoc

```diff
@@ -13,5 +15,90 @@ As the primary mechanism for traffic to enter the cluster, the demands on the In
 * Allow certain Ingress Controllers to have different policies defined.
 * Allow only specific routes to use additional features.
 * Expose different routes on different addresses so that internal and external users can see different routes, for example.
+* Transfer traffic from one version of an application to another during a blue green deployment.
 
-Ingress Controller can use either route labels or namespace labels as a sharding method.
+When Ingress Controllers are sharded, a given route is bound to zero or more routers in the group. The route binding ensures uniqueness of the route across the shard. Uniqueness allows secure and non-secure versions of the same route to exist within a single shard. This implies that routes now have a visible life cycle that moves from created to bound to active.
+
```

Comment:
Also, related to the comment on uniqueness, we noticed that the idea of route "uniqueness", though not specific to sharding, it not really explained anywhere in the documentation. The idea of route uniqueness is that a route is unique if has one of characteristics different:
1. TLS Termination
2. Hostname
3. Path

I don't think this concept is address in documentation, but I think that's probably something we can address later and not in this PR.

Reply:
@gcs278 thanks for this, do you have more information on the topic? I can create a JIRA ticket to address this as a continuous improvement task (CIT) now that we're post GA 🙂

Reply:
Sounds good, I think that's about it, but if you make the Jira Ticket, I can help fill in any holes (or I can get Miciah to if I can't).